.zshrc

```diff
@@ -62,9 +62,7 @@ function zz() {
   local to=`{
     echo $HOME/.files;
     fd --full-path $HOME/src --type d --max-depth=1 --absolute-path $HOME/src;
```

Comment:
yup, it's better with `/src` since macos has own structure in **my** home folder...